### DISCUSSION TITLE
OAuth provider – Yandex.Connect

### DISCUSSION THREAD

**Discussion by asn007** [outline/outline#3517] [Ideas]
Hey there!
At my workplace we're using Yandex.Connect for provisioning corporate accounts, and we'd like integration with it to be added

I can implement it, but I'd like the feedback - would you accept such a PR into mainline? Having a separate fork and having to continuously merge changes from mainline would be a hassle, hence the question

**Comment by tommoor** [comment]
We're not accepting additional authentication providers at this time.

Is Yandex not OIDC compatible?
